keygen/tarsnap-keygen.1-mdoc

```diff
@@ -10,9 +10,9 @@
 .Xr tarsnap 1
 .Sh SYNOPSIS
 .Nm
-.Fl -keyfile Ar key-file
-.Fl -user Ar user-name
-.Fl -machine Ar machine-name
+.Fl -keyfile Ar filename
```

Comment:
I'd like to stick with `key-file`.  And writing

> The --keyfile **key-file** option specifies the name of the file in which to write the newly-generated keys.

seems entirely grammatical...?